manifests/v1beta1/components/controller/rbac.yaml

```diff
@@ -62,6 +62,12 @@ rules:
       - taskruns
     verbs:
       - "*"
+  - apiGroups:
```

Comment:
If users are not using Argo, they’ll have to add these permissions? I’d recommend adding these to a separate file to help users who would like to use Argo

Reply:
The other operators should be fine since they are part of Kubeflow org.

Reply:
@terrytangyuan @johnugeorge Do you think it's better to add steps for updating Katib controller and RBAC to appropriate README. For example, for Argo: https://github.com/kubeflow/katib/blob/8acbffcab5ce8d74742967a8d3665ab72faf351c/examples/v1beta1/argo/README.md and for Tekton: https://github.com/kubeflow/katib/tree/master/examples/v1beta1/tekton

Reply:
Yes I think that should be fine.

Reply:
SGTM